packages/trader/src/AppV2/Containers/ContractDetails/contract-details.scss

```diff
@@ -0,0 +1,52 @@
+.contract-details {
+    background-color: rgb(245, 247, 249);
+    padding: 8px;
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+    height: 100vh;
+    gap: var(--semantic-spacing-general-sm);
+    padding-bottom: 70px;
```

Comment:
no need for this. 8px for the container all the way around and set `height: 100%` will fix.